### PR TITLE
Refactor ItemAttributes.cs

### PR DESCRIPTION
**Pull request by RedlineTriad**
### Purpose
Refactors ItemAttributes to get rid of unnecessary fields, assignments, and comment the code better<br>

### Open Questions and Pre-Merge TODOs

- [x]  This fix is tested on the same branch it is PR'ed to.
- [x]  I correctly commented my code
- [x]  My code is indented with tabs and not spaces
- [x]  This PR does not include any unnecessary .meta, .prefab or .unity (scene) changes
- [x]  This PR does not bring up any new compile errors
- [x]  This PR has been tested in editor
- [ ]  This PR has been tested in multiplayer (with 2 clients and late joiners, if applicable)

### CODE REVIEW

**Comment by fomalsd** [comment]
Thanks! The only thing I'm worried about is whether any of our prefabs used custom values for removed fields

**Comment by RedlineTriad** [comment]
Almost all of the removed fields were private

**Comment by RedlineTriad** [comment]
The only public field that was removed was `type` which was set to `itemType` on construction so the only way this could be a problem was if after being created, they changed type.
If this is the case, I am pretty sure we want it to burn down as quickly as possible so the mistake is found.
Though I haven't found any errors in the editor.

**Comment by fomalsd** [comment]
yes, thankfully itemType is correct in all prefabs I've found, so it's all good!

File: UnityProject/Assets/Scripts/Items/ItemAttributes.cs [hunk 1]
```diff
@@ -415,72 +423,64 @@ private static ItemSize getItemSize(string s)
...
-		{
-			UIManager.SetToolTip = itemName + " (" + itemDescription + ")";
-		}
-		else
-		{
-			UIManager.SetToolTip = itemName;
+		string tooltip = itemName;
+
+		if (!String.IsNullOrEmpty(itemDescription)) {
+			itemName += $" ({itemDescription})";
```
Comment (fomalsd) [inline UnityProject/Assets/Scripts/Items/ItemAttributes.cs]: Don't append item's description to its name:
image

**Review by fomalsd — CHANGES_REQUESTED** [review]
Found a bug with `itemName`:

**Review by fomalsd — APPROVED** [review]

**Pull request merged**